File: GitNexus/gitnexus/test/fixtures/lang-resolution/swift-method-enrichment/Sources/Animal.swift
protocol Animal {
    func speak() -> String
}

class Dog: Animal {
    func speak() -> String {
        return "woof"
    }

    static func classify(_ name: String) -> String {
        return "mammal"
    }

    @objc final func breathe() -> Bool {
        return true
    }
}

File: GitNexus/gitnexus/test/fixtures/lang-resolution/javascript-method-enrichment/animal.js
class Animal {
    speak() {
        throw new Error("Not implemented");
    }

    static classify(name) {
        return "mammal";
    }

    breathe() {
        return true;
    }
}

class Dog extends Animal {
    speak() {
        return "woof";
    }
}

module.exports = { Animal, Dog };
